### ISSUE TITLE
What is the release cyle and development branch model of this project? 

### ISSUE THREAD

**Issue by talha131** [chentsulin/electron-react-boilerplate#395]
The [official documentation](https://github.com/chentsulin/electron-react-boilerplate#install) recommends to get this boilerplate from master branch. 

```git
git clone https://github.com/chentsulin/electron-react-boilerplate.git your-project-name
```

But this may not always work. See #393  and #391. 

There should be a separate develop branch. So that those of use who fetch from master get only the release builds of the boilerplate.

This project does have [release tags](https://github.com/chentsulin/electron-react-boilerplate/releases), which is very cool. But [last release](https://github.com/chentsulin/electron-react-boilerplate/releases/tag/v0.10.0) was in April 18. 

Since then around [113 changes](https://github.com/chentsulin/electron-react-boilerplate/compare/v0.10.0...master) have been made in the project that have not been released yet.

Bottom line is

1. Please update release tag 
1. Please move development to separate branch OR update the documentation

**Comment by amilajack** [comment]
Great idea! Thanks for the reminder

**Comment by amilajack** [comment]
Merged into https://github.com/chentsulin/electron-react-boilerplate/issues/404
